GLOSSARY.md

```diff
@@ -45,6 +45,48 @@ Class names and proper nouns do not get translated.
 - protected (function) - защищённая (функция)
 - public (function) - общедоступная (функция)
 - rebind variable - переприсвоение переменной
-- substraction - вычитание
+- subtraction - вычитание
 - signature - сигнатура (функции)
 - tail - хвост
+
+
+### Glossary (portuguese)
+
+#### Elixir-related terms
+
+- actors - atores
+- atom - átomo
+- function clauses - cláusulas de função
+- guard - guarda
+- guard operator - operador de guarda
+- keyword list - lista de palavras
+- match operator - operador para procura de padrões
+- pin operator - operator para apontar para último valor atribuido a variável
+- pipe operator - operador pipe
+- sigils - tag de padrão
```

Comment:
:+1: , the official documentation says: `sigils are one of the mechanisms provided by the language for working with textual representations`

I would say: `mecanismos para transformação de representações textuais` Once it's created to literal representations.